canonical-websites/maas.io
# Tour updates

## Done

- Creates new update to the tour page, including new screen shots / content / accessible markup
- New two tone row
- Removes un required JS

## QA

- Pull down branch
- Run ```npm i``` and ```make develop```
- View page at http://localhost:8000/tour
- Run ```pa11y http://localhost:8000/tour``` if installed and check for no errors

## Issue / Card

Designs https://github.com/CanonicalLtd/MAAS-design/issues/99
Content https://docs.google.com/document/d/1KcYwNK0pJyiDVEDq4vTpv6BX_rpSDnxUxtAvAnsRKyw/edit

Fixes

#86 #96 #95 #98 

## Screenshots

_n/a_
